### DISCUSSION TITLE
Esp32-cam Ai Thinker and homeassistant - Haade.fr

### DISCUSSION THREAD

**Discussion by giscus** [haade-administrator/haade.fr#33] [Comments]
# Esp32-cam Ai Thinker and homeassistant - Haade.fr

Complete and easy installation of the Esp32-cam Ai-thinker camera in homeassistant via esphome

https://haade.fr/en/blog/esp32cam-ai-thinker-homeassistant-esphome-full-configuration

**Comment by AdamM68** [comment]
super !! :)
but ! on the picture of FTDI there is 3.3v , You write that we should use 5v.  What is the truth ?  I think esp32 should use 3.3v !
